### PR TITLE
 week 3 homework is added

### PR DESCRIPTION
**Pull request by samima55**

### CODE REVIEW

File: javascript/javascript1/week3/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>week3</title>
+    <script src="index.js"></script>
```
Comment (naresh699) [inline javascript/javascript1/week3/index.html]: please read why and where should keep js either in the head or at end of the body.

File: javascript/javascript1/week3/index.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+// removing name 
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
```
Comment (naresh699) [inline javascript/javascript1/week3/index.js]: we dont need ',' in the last index of the array.

File: javascript/javascript1/week3/index.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+  function logOutSeriesText(seriesDurations) {
+       let totalPercentage=0
+    for(let i=0; i<seriesDurations.length; i++){
+      let individualTime =seriesDurations[i].days
+      +seriesDurations[i].hours/24
+      +seriesDurations[i].minutes/1440;
+      let percentage=(individualTime/365)/(80);
+      totalPercentage +=percentage;
+      console.log(seriesDurations[i].title +" took " +percentage.toFixed(5) +" of my life");
```
Comment (naresh699) [inline javascript/javascript1/week3/index.js:74]: in fresh code try to use 'template string' for adding variables in output.

File: javascript/javascript1/week3/index.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    }
+  }
+}
+const firstNote = getNote(1);
+console.log(firstNote);
+
+
+function logOutNotesFormatted(){
+    for(const item of notes){
+        console.log(`\n note with id : ${item.id}, has following note:
```
Comment (naresh699) [inline javascript/javascript1/week3/index.js:110]: great, this template string is expected in other places also. ie  #76

**Review by naresh699 — COMMENTED** [review]
- js should be at the end of the body.
- maybe in git, but the formatting of js is not proper.
- I am seeing many comments in js for explaining the use of the script. read the JSDOC tutorial. this will give you inside about commenting in js

File: javascript/javascript1/week3/index.html (hunk 1 above)
Comment (samima55) [reply javascript/javascript1/week3/index.html]: i have modified it locally but if i want to commit and push these changes do i need to make the branch js/js/week3 again ? or i just need to add commit and push after modifying ?

**Pull request merged**